src/app/query/common/compliance-checker/compliance-checker.query.ts

```diff
@@ -15,6 +15,8 @@ const headers = {
   Token: '51d3c7529eb08a8c62d41d70d006bdcd4248150fbd6826d5828ac908e7c12073',
 };
 
+export const isComplianceCheckEnabled = false;
```

Comment:
shouldn't this be added to config?

Reply:
I think it should be the idea here is to quickly disable this for release and then re-enable afterwards so lets let it slide this time